assets/home/tasks-section/task-item.js

```diff
@@ -10,15 +10,21 @@ import CheckIcon from '../../icons/checked.svg';
 import ChevronRightIcon from '../../icons/chevron-right.svg';
 import { isUrlExternal } from '../utils';
 
+/**
+ * WordPress dependencies
+ */
+import { Icon, external } from '@wordpress/icons';
+
 /**
  * Tasks item component.
  *
- * @param {Object}  props       Component props.
- * @param {string}  props.title Item title.
- * @param {string}  props.url   Item URL.
- * @param {boolean} props.done  Whether item is completed.
+ * @param {Object}  props                  Component props.
+ * @param {string}  props.title            Item title.
+ * @param {string}  props.url              Item URL.
+ * @param {boolean} props.done             Whether item is completed.
+ * @param {boolean} props.showExternalIcon Whether to show external icon.
  */
-const TaskItem = ( { title, url, done } ) => {
+const TaskItem = ( { title, url, done, showExternalIcon } ) => {
```

Comment:
While testing it, I realized another thing. Instead of having the new property, shouldn't we use the `isExternal` variable we have here?

I noticed it because I think we should also use the same `linkProps` in the current case. And I suspect it would also make sense to apply the icon for the other external links, right?

Reply:
Ah, I think it wouldn't work because it will check our internal API link. But Maybe we could expand that conditional adding a "special mark" in the URL, like `&external=true` or something like this. WDYT?

Reply:
Yap, this is the first idea that I came up with when implementing this. But then I decided not to do it that way because this kind of works like a magic logic, whereas having a property with documentation on top makes it much easily understandable. It can be hard for a future dev to understand what to do to show an external link unless they read the internal logic of the component, but when there's a documented param, it makes it reusable.

But after you also recommended, and I looked at the amount of code needed to do just that, I decided to use the special URL approach https://github.com/Automattic/sensei/pull/7562/commits/2ae3cdd2137e0b20085189c1c44f0a9d6cc33673 . To make it more understandable in the future, I've added a comment here https://github.com/Automattic/sensei/pull/7562/commits/fc5fa71e275ef8c47619374e716861c298ba237c.

Thank you!